node/src/actor_system_factory.rs

```diff
@@ -1320,7 +1322,7 @@ mod tests {
         let _ = subject.prepare_initial_messages(
             make_cryptde_pair(),
             config.clone(),
-            Box::new(PersistentConfigurationMock::new()),
+            Box::new(PersistentConfigurationMock::new().set_min_hops_count_result(Ok(()))),
```

Comment:
It really surprises me that tests at this level (We are in actor_system_factory.rs!!) ask about results for setting the min hops count. This seems wrong. Investigate whether, why and where...

My understanding is that the `set_min_hops_count()` method ought to be used only in the function where various values are set from the BootstrapperConfig into the database. That should happen when the config is complete and I'm pretty sure entering the actor system factory domain means we've done with the setup and the config is complete. 

In other words, that method should definitely be called outside of this space. When you are already here you should be able to read everything you need to know from the bootstrapper_config because that's exactly its purpose.

Therefore I hope this is just a mistake and you can remove this results (in other tests too) and they will keep running fine. If that's not true, you will have to reconsider whether the code is okay.

Reply:
Done 👍 

This is just a residue of old code for this card.

Previously, I was setting up the value while starting the actors. This has now changed and we're setting up the value in `configure_database`.

I was able to remove this code as if it was a dead leaf of tree.

Reply:
The answer is poetic. Nice :)